build-tools/packages/build-cli/docs/merge.md

```diff
@@ -12,12 +12,9 @@ Sync branches depending on the batch size passed
 
 ```
 USAGE
-  $ flub merge branches -a <value> -p <value> -s <value> -t <value> -b <value> --reviewers <value> [-v | --quiet]
-    [-r <value>]
+  $ flub merge branches -p <value> -s <value> -t <value> -b <value> --reviewers <value> [-v | --quiet] [-r <value>]
 
 FLAGS
-  -a, --auth=<value>       (required) GitHub authentication token. For security reasons, this value should be passed
```

Comment:
To confirm, you still have to provide a PAT, right? Can the PAT be provided using an env variable? If not, we should add that. For security reasons, you generally don't want to pass a token via the CLI, because terminals have history, and your PAT winds up in the terminal logs, which can disclose your token unexpectedly. With an env variable, what people usually do is put the token in a file, then read that file into an env variable, and read the token from the variable.

To be clear, this is only relevant for local testing. The bot/workflow is fine. But supporting an env variable will make local testing much more secure.

Reply:
I did setup the environment variables using the env keyword in GitHub Actions which will not expose the values in the terminal logs by default.